docs/design-patterns.mdx

```diff
@@ -2,9 +2,13 @@
 title: Cadence Design Patterns
 ---
 
-This is a selection of software design patterns developed by core Flow developers while writing Cadence code for deployment to Flow Mainnet.
+This is a selection of software design patterns developed by core Flow developers
+while writing Cadence code for deployment to Flow Mainnet.
 
-[Design patterns](https://en.wikipedia.org/wiki/Software_design_pattern) are building blocks for software development. They may provide a solution to a problem that you encounter when writing smart contracts in Cadence. Remember that if they do fit they may not be the right solution for a given situation, and they are not meant to be a set of prescriptions to be followed exclusively even where a better solution represents itself.
+[Design patterns](https://en.wikipedia.org/wiki/Software_design_pattern) are building blocks for software development.
+They may provide a solution to a problem that you encounter when writing smart contracts in Cadence.
+Remember that if they do fit they may not be the right solution for a given situation,
+and they are not meant to be a set of prescriptions to be followed exclusively even where a better solution represents itself.
```

Comment:
copy edit as this feels clunky to me. How about:

_If they do not clearly fit, these patterns may not be the right solution for a given situation or problem.
They are not meant to be rules to be followed strictly, especially where a better solution presents itself._